macros/Embed_text.ejs

```diff
@@ -26,14 +26,14 @@ var m2 = mdn.localString({
 if (brush.indexOf($1) > -1) {
     syntaxe = $1;
 }
-  
-env.files.forEach(function(aFile) {
+
+for(let aFile of env.files) {
```

Comment:
Nit: Our code standard here (not properly documented yet ;() requires a space between `for` and the opening parentheses. I believe that is what Prettier does as well? @davidflanagan

Reply:
I've always written them without spaces. But prettier inserts them, so we're good on that account. 

If we take Florian at his word, the macros are going to be eliminated, so I don't think it is worth taking the time to make them look nice.  (At least not in this PR!)